### PR TITLE
Add content to on prem page

### PR DESCRIPTION
**Pull request by deadlight**
## Done
Added content to the On Prem page

## QA
 - ./run
 - Visit /set-up-on-prem
 - Check at various viewports

### CODE REVIEW

**Comment by anasereijo** [comment]
@deadlight I think the title should be `h1`. Everything else looks fine!

File: templates/landscape/set-up-on-prem.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{% block description %}Landscape is the leading management tool to deploy, monitor and manage your Ubuntu servers.{% endblock %}
+
+{% block content %}
+<section class="p-strip">
+  <div class="row">
+    <h1>Set up your on premises Landscape server</h1>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <ol class="p-list-step">
```
Comment (anthonydillon) [inline templates/landscape/set-up-on-prem.html]: What about using `p-stepped-list--detailed`?

**Review by anthonydillon — APPROVED** [review]
Looks good but a recommendation in code.

**Comment by anthonydillon** [comment]
Also what about the copy to clipboard? https://docs.vanillaframework.io/en/patterns/code-snippet

**Comment by deadlight** [comment]
@anthonydillon @anasereijo Updated

**Comment by anthonydillon** [comment]
Very nice :+1:

**Pull request merged**